extensions/wikia/Email/EmailHelper.class.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Email;
+/**
+ * Class Helper
+ *
+ * Common methods used between controllers and classes
+ *
+ * @package Email
+ */
+class Helper {
+	const REQUIRED_USER_RIGHT = 'access-sendemail';
```

Comment:
Can either the SpecialSendEmailController use the constant from this class, or vice-versa, instead of having it in two places?